tests/results/DiffTest.py

```diff
@@ -138,10 +138,10 @@ def test_modified(self):
         self.uut.change_line(3, '3', '3.changed')
 
         self.assertEqual(self.uut.modified, result_file)
-        self.assertEqual(self.uut.original, self.file)
 
         self.uut.delete_line(len(self.file))
-        del result_file[len(result_file) - 1]
+        del result_file[-1]
+        result_file[-1] = result_file[-1].rstrip('\n')
```

Comment:
ah this is quite an edge case ... if the old version has no \n's and we delete the last line... though not really important for now, we should maybe document that case more properly later or change behaviour, don't know ;)